### PR TITLE
OPS-2796 Changed login pages to new domains

### PR DESCRIPTION
**Pull request by dimapin**
# Short Description

As the Brandenburg and the HPI Instances are moved to new domains the presented login pages need also a change

## Links to Ticket and related Pull-Requests
https://ticketsystem.hpi-schul-cloud.org/browse/OPS-2796

### CODE REVIEW

File: locale/de.json [hunk 1]
```diff
@@ -619,11 +619,11 @@
 	"pages.courses.new.btn_new": "Erstelle neuen Kurs",
 	"pages.courses.new.title": "Neuen Kurs erstellen",
 	"pages.impressum.title": "Impressum",
-	"pages.loginInstances.brb_link": "https://brandenburg.schul-cloud.org/login",
+	"pages.loginInstances.brb_link": "https://brandenburg.cloud/login",
 	"pages.loginInstances.bundesland": "Wähle deine Cloud",
-	"pages.loginInstances.hpi_link": "https://hpi-schul-cloud.de/login",
+	"pages.loginInstances.hpi_link": "https://dbildungscloud.de/login",
```
Comment (monaweitzenberg) [inline locale/de.json:624]: the key should be renamed as well like
"pages.loginInstances.dbildungscloud_link

File: locale/en.json [hunk 2]
```diff
@@ -613,11 +613,11 @@
 	"pages.courses.new.btn_new": "Create",
 	"pages.courses.new.title": "Create new course",
 	"pages.impressum.title": "Imprint",
-	"pages.loginInstances.brb_link": "https://brandenburg.schul-cloud.org/login",
+	"pages.loginInstances.brb_link": "https://brandenburg.cloud/login",
 	"pages.loginInstances.bundesland": "In which state would you like to log in?",
-	"pages.loginInstances.hpi_link": "https://hpi-schul-cloud.de/login",
+	"pages.loginInstances.hpi_link": "https://dbildungscloud.de/login",
```
Comment (monaweitzenberg) [inline locale/en.json:618]: the key should be renamed as well like
"pages.loginInstances.dbildungscloud_link

File: locale/es.json [hunk 3]
```diff
@@ -627,11 +627,11 @@
 	"components.molecules.VTaskItemTeacher.submitted": "Entregado",
 	"components.molecules.VTaskItemTeacher.graded": "Calificado",
 	"pages.impressum.title": "Impresión",
-	"pages.loginInstances.brb_link": "https://brandenburg.schul-cloud.org/login",
+	"pages.loginInstances.brb_link": "https://brandenburg.cloud/login",
 	"pages.loginInstances.bundesland": "¿En qué provincia le gustaría iniciar sesión?",
-	"pages.loginInstances.hpi_link": "https://hpi-schul-cloud.de/login",
+	"pages.loginInstances.hpi_link": "https://dbildungscloud.de/login",
```
Comment (monaweitzenberg) [inline locale/es.json:632]: the key should be renamed as well like
"pages.loginInstances.dbildungscloud_link

File: locale/de.json (hunk 1 above)
Comment (dimapin) [reply locale/de.json]: The Key is also aligned with new domain

File: locale/en.json (hunk 2 above)
Comment (dimapin) [reply locale/en.json]: The Key is also aligned with new domain

**Review by monaweitzenberg — APPROVED** [review]
Thx for your work! :)

**Review by CeEv — APPROVED** [review]

**Pull request merged**